src/main/cpp/t06_longest.cpp

```diff
@@ -17,10 +17,36 @@
 
 #include "t06_longest.h"
 #include <iostream>
+#include <string>
 
-
-using namespace std;
-
+std::string blabla(std::string line){
+    int curr_len = 0;
+    int max_len = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'max_len' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4718331)